ethportal-api/README.md

```diff
@@ -46,5 +46,18 @@ async fn main() {
 }
 ```
 
+## Types
```

Comment:
Nice! Maybe it's worth updating to `## Types and Utils`? Technically we did move the hex utils into this crate and it seems to me like the same convention is worth adopting for shared utils